### PR TITLE
[Cherry-Pick]【TI-Consisent】Added Metric logits_stats to the ZMQ branch (#6979)

### PR DESCRIPTION
**Pull request by liuruyan**
## Motivation

**背景**：出于对训推一致性丰富检测指标及长期CI/CE监控考量，对sample之后的logits添加logits_stats(min/max/mean/std)，用来保证确定性及稳定性。

## Modifications

**数据结构及接口**：由于logprob与logits_stat同样是输出的重要检测指标且均从logits计算而来，暂时实现先将logits_stat存入LogprobsTensors数据结构保存，并升级logprob传播链路上相关接口，支持同时透传logits_stats
  ```python
  class LogprobsTensors(NamedTuple):
      """ """
  
      # [num_reqs, max_num_logprobs + 1]
      logprob_token_ids: paddle.Tensor
      # [num_reqs, max_num_logprobs + 1]
      logprobs: paddle.Tensor
      # [num_reqs]
      selected_token_ranks: paddle.Tensor
      # Logits statistics for each sequence (optional)
      logits_min: Optional[paddle.Tensor] = None  # [num_reqs]
      logits_max: Optional[paddle.Tensor] = None  # [num_reqs]
      logits_mean: Optional[paddle.Tensor] = None  # [num_reqs]
      logits_std: Optional[paddle.Tensor] = None
      ...
```
**FLAG**：添加与enable_logprob同级别model_config：`self.compute_logits_stats = False`，且在server启动时支持配置`--compute-logits-stats`

> 注：由于改变了返回字段，导致有些单测无法通过，所以改动单测文件，返回值中提出新增字段（logits_stats ）

## Usage or Command
- 本功能暂时只支持ZMQ，流式与非流式测试均可正常返回
- 启动FD服务时需要同时开启`--compute-logits-stats`,`--enable-logprob`
```bash
export FD_USE_GET_SAVE_OUTPUT_V1=1 
python -m fastdeploy.entrypoints.openai.api_server \
       --enable-logprob \
       --compute-logits-stats \
       ...  # more setting
```
- 发送请求时需要指定`logprobs=True`,`top_logprobs=0`
```bash
response = client.chat.completions.create(
    model="null",
    messages=[
        {"role": "system", "content": "I'm a helpful AI assistant."},
        {"role": "user", "content": "把李白的静夜思改写为现代诗"},
    ],
    stream=True,  # False
    max_tokens=100,
    logprobs=True,
    top_logprobs=0
)
```

## Accuracy Tests

本PR不涉及精度修改，稳定性测试结果如下：
1. server 实例
```
export CUDA_VISIBLE_DEVICES=5
export FD_USE_GET_SAVE_OUTPUT_V1=1
python -m fastdeploy.entrypoints.openai.api_server \
       --tensor-parallel-size 1 \
       --load-choices "default_v1" \
       --port 8137 \
       --engine-worker-queue-port "8978" \
       --metrics-port 8977 \
       --max-model-len 8192 \
       --max-num-seqs 32 \
       --gpu-memory-utilization 0.9 \
       --reasoning-parser ernie5 \
       --quantization "block_wise_fp8" \
       --model "$MODEL_PATH" \
       --graph-optimization-config '{"use_cudagraph":true}' \
       --no-enable-prefix-caching \
       --enable-logprob \
       --compute-logits-stats \
```
2. 流式请求
```
import openai
host = "0.0.0.0"
port = "8137"
response = client.chat.completions.create(
    model="null",
    messages=[
        {"role": "system", "content": "I'm a helpful AI assistant."},
        {"role": "user", "content": "把李白的静夜思改写为现代诗"},
    ],
    stream=True,
    max_tokens=100,
    seed=0,
    top_p=1e-5,
    logprobs=True,
    top_logprobs=0
)
for chunk in response:
    print(chunk)
    if chunk.choices[0].delta:
        print(chunk.choices[0].delta.reasoning_content, end='')
print('\n')
```
3. 非流式请求
```
import openai
host = "0.0.0.0"
port = "8137"
client = openai.Client(base_url=f"http://{host}:{port}/v1", api_key="null")
response = client.chat.completions.create(
    model="null",
    messages=[
        {"role": "system", "content": "I'm a helpful AI assistant."},
        {"role": "user", "content": "把李白的静夜思改写为现代诗"},
    ],
    stream=False,
    max_tokens=100,
    seed=0,
    top_p=1e-5,
    logprobs=True,
    top_logprobs=0
)
print(response)
```
4. 实验结果：
- 各自多次实验所有Token logits指标均可对齐
- 流式及非流式所有Token logits指标均可对齐

- 流式request返回体：
image

- 非流式request返回体：
image

## Checklist

- [ ] Add at least a tag in the PR title.
  - Tag list: [`[FDConfig]`,`[APIServer]`,`[Engine]`, `[Scheduler]`, `[PD Disaggregation]`, `[Executor]`, `[Graph Optimization]`, `[Speculative Decoding]`, `[RL]`, `[Models]`, `[Quantization]`, `[Loader]`, `[OP]`, `[KVCache]`, `[DataProcessor]`, `[BugFix]`, `[Docs]`, `[CI]`, `[Optimization]`, `[Feature]`, `[Benchmark]`, `[Others]`, `[XPU]`, `[HPU]`, `[GCU]`, `[DCU]`, `[Iluvatar]`, `[Metax]`]
  - You can add new tags based on the PR content, but the semantics must be clear.
- [ ] Format your code, run `pre-commit` before commit.
- [ ] Add unit tests. Please write the reason in this PR if no unit tests.
- [ ] Provide accuracy results.
- [ ] If the current PR is submitting to the `release` branch, make sure the PR has been submitted to the `develop` branch, then cherry-pick it to the `release` branch with the `[Cherry-Pick]` PR tag.

### CODE REVIEW

**Review by EmmonsCurse — APPROVED** [review]
LGTM～ Skip coverage check as it mainly relies on end-to-end tests.

**Review by sunlei1024 — APPROVED** [review]

**Review by zoooo0820 — APPROVED** [review]
